packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -597,13 +602,18 @@ export function validateBlobsAndTrees(snapshot: IOdspSnapshot): void {
 	);
 }
 
-function countTreesInSnapshotTree(snapshotTree: ISnapshotTree): number {
-	let numTrees = 0;
-	for (const [_, tree] of Object.entries(snapshotTree.trees)) {
-		numTrees += 1;
-		numTrees += countTreesInSnapshotTree(tree);
+function getTreeStatsCore(snapshotTree: ISnapshotTree, stats: ITreeStats): void {
+	stats.blobNodes += Object.entries(snapshotTree.blobs).length;
+
+	const entries = Object.entries(snapshotTree.trees);
+	if (entries.length === 0) {
```

Comment:
entries is not being reused, so can just write 
```if(Object.entries(snapshotTree.trees).length === 0)```

Reply:
Yes, that was intention but somehow it was made only in my head :)